nss-day-cohort-45/hello-world-z-indexes
# added info about Greece and Ireland

## Added Image File with images & added info about greece and Ireland in index.html

## steps to test
1.  Check to see if you have an image folder
2. Check if 2 images are in the folder - one of ireland and one of greece
3.  Check info added about greece and ireland
